banzai/images.py

```diff
@@ -79,6 +79,10 @@ def __init__(self, runtime_context, filename=None, data=None, data_tables=None,
             self.gain = eval(str(header.get('GAIN')))
 
         self.ccdsum = header.get('CCDSUM')
+        self.configuration_mode = header.get('CONFMODE', file_utils.ccdsum_to_filename(self.ccdsum))
```

Comment:
For clarity's sake, make the default value 'N/A'

Reply:
Done.